lib/page/countries.rb

```diff
@@ -0,0 +1,18 @@
+require 'everypolitician'
+require_relative '../world'
+
+module Page
+  class Countries
+    def countries
+      @countries ||= EveryPolitician.countries
+    end
+
+    def download_url
+      @download_url ||= EveryPolitician.countries_json
+    end
+
+    def world
+      @world ||= World.new.as_json.to_a
```

Comment:
`World.new.as_json` returns a Hash, so it's a little odd to coerce it to an Array. The only thing that the template does with this at the moment is call `.length` on it, which gets the right thing either way, but if the template tried to do anything extra with this it could get quite confusing. 

We'll probably want a follow-up PR to remove that logic entirely from the template, and only provide a `missing_country_count` method here instead, but until we do that I think I'd be a bit more comfortable if we just passed the Hash around here instead.